services/addons/images/firmware_manager/download_blob.py

```diff
@@ -14,3 +14,13 @@ def download_gcp_blob(blob_name, destination_file_name):
     logging.info(
         f"Downloaded storage object {blob_name} from bucket {bucket_name} to local file {destination_file_name}."
     )
+
+
+# "Download" a blob from a directory mounted as a volume in a Docker container
+def download_docker_blob(blob_name, destination_file_name):
+    directory = "/mnt/blob_storage"

```

Comment:
Please follow standard comments style in function, and provide some details about the parameters and possible hints of the parameter types.

Reply:
Single-line comments have been replaced with docstrings in `download_blob.py`